app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
It is a good practice to validate the input before processing it. In this case, you should check if the `command` is a string and if it is not empty.